R/data.R

```diff
@@ -218,3 +218,30 @@
 #'
 "monthly_prey_density"
 
+#' Steve Zeug San Joaquin Data
+#'
```

Comment:
It would be good to add some additional documentation here for each of the authors. Email, links to  source, and short description of the data structure/format